docs/notebooks/survival_model.ipynb

```diff
@@ -0,0 +1,1587 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    lam = 1/np.exp(az.summary(idata)["mean"]["Intercept"])
</pre><p><code>az.summary()</code> is being called multiple times. We could call it once and store the result in a variable such as <code>idata_summary</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>